src/create_disk.sh

```diff
@@ -142,6 +142,10 @@ case "$arch" in
         sgdisk -p "$disk"
         ;;
     s390x)
+        # NB: in the bare metal case, this partition table is not what actually
+        # gets written to disk in the end: coreos-installer has code which
+        # transforms it into a DASD-compatible partition table and copies each
+        # partition individually bitwise.
```

Comment:
That's only true when installing onto a DASD, not an FBA disk.

Reply:
Clarified.